Exercises_2023.md

```diff
@@ -41,14 +44,17 @@ _To prepare for week 3,_ a few big scientific collaboration ([see list here](Cou
     + In pairs, the students identify the jargon in their layperson summaries, and discuss how to avoid it in the science communication (inspiration from upgoer 5)
     + Summaries will be tested using language level tools (e.g. [textstat](https://github.com/textstat/textstat), tbd)
 
+### To prepare for week 7
+Watch the [lecture of prof. Joshua Pearce](https://uwoca-my.sharepoint.com/:v:/g/personal/jpearc26_uwo_ca/EXHKiseU0ExGjcIuwGHwZUYBfB5tD74PdjOI_ba428V4bw) to be prepared for the discussions and questions during the class
```

Comment:
Is it ok to put this (personal) link on a public site?

Reply:
I asked his permission and he was okay. It would be best if he puts it on YT, but I was not sure if he wants to use his personal channels. maybe we should have our own OS4P YT account. should we check with students?